source/chapters/1/intro.md

```diff
@@ -4,37 +4,72 @@ description: Getting started with Meteor
 disqusPage: 'Chapter 1: Intro'
 ---
 
-## Installing
+## Installing the framework
+In order to be able to develop with Meteor, you will need three basic components: Nodejs, NPM and the Meteor framework itself.
+So, to get what you need, run the following command on your system terminal:
 
-https://www.meteor.com/install
+```
+curl https://install.meteor.com/ | sh
+```
+
+This command will work on an Ubuntu based distribution, such as Linux Mint, Elementary OS and so on.
+This will also work on a Mac!
+To check that you really have installed what you needed, check the software versions of the components:
+```
+nodejs -v
+npm -v
+meteor --version
+```
+If you don't have errors after running these commands, congratulations ! You can move on ! 
+However, if you have erros, if you ran the commands correctly. 
+Afterwards, Google is your best friend ! 
+
+On Windows, simply download the installer and follow the basic Windows installation procedure that most software has.
+Here, instead of Bash,Terminator or iTerm, you can use Powershell or the good old Command Prompt.
```

Comment:
For Powershell leave a link that opens in a new tab. Read about "markdown links"